xdr/muxed_account.go

```diff
@@ -21,6 +21,43 @@ func MustMuxedAddressPtr(address string) *MuxedAccount {
 	return &muxed
 }
 
+func MuxedAccountFromAccountId(gAddress string, id uint64) (MuxedAccount, error) {
+	bytes, err := strkey.Decode(strkey.VersionByteAccountID, gAddress)
+	if err != nil {
+		return MuxedAccount{}, err
+	}
+
+	var publicKey Uint256
+	copy(publicKey[:], bytes)
+
+	muxed, err := NewMuxedAccount(
+		CryptoKeyTypeKeyTypeMuxedEd25519,
+		MuxedAccountMed25519{
+			Id:      Uint64(id),
+			Ed25519: publicKey,
+		},
+	)
+	if err != nil {
+		return MuxedAccount{}, err
+	}
+
+	keyBytes, err := muxed.Med25519.Ed25519.MarshalBinary()
+	if err != nil {
+		return MuxedAccount{}, err
+	}
+
+	idBytes, err := muxed.Med25519.Id.MarshalBinary()
+	if err != nil {
+		return MuxedAccount{}, err
+	}
+
+	mAddress := strkey.MustEncode(
+		strkey.VersionByteMuxedAccount,
+		append(keyBytes, idBytes...),
+	)
+	return AddressToMuxedAccount(mAddress)
```

Comment:
I think I'm missing something here but why can't we return muxed instead of all these lines? `strkey` roundtrip seems unnecessary?
```suggestion
	return muxed, err
```

Reply:
I must've been written this at 3am or something... that's the only explanation. That or working with muxed account objects is just *confusing*, hence the need for said helpers lol. Anyway, tysm for the catch, resolved in latest push.